### PR TITLE
Dev screpyy96 skills section done

### PR DESCRIPTION
**Pull request by screpyy96**

### CODE REVIEW

File: src/_component/_skills/skills.pug [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+                class=`skills__skill--${skill.type}`
+            )=skill.label
+.column(span="row" height="10")
+.column(span="row")
+    .about-box 
+        p.description= data['about-me'].descriptionList
+           
+
+
+
```
Comment (RalucaRbc) [inline src/_component/_skills/skills.pug:32]: Oppulance 😆  You can delete some of these empty lines and leave just 1 😄

File: src/_component/_skills/skills.scss [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  list-style-type: none;
+}
+
+.skills__item {
+  display: flex;
+  border: 2px solid #707070;
+
+  &:first-child .skills__icon {
+    color: #d52027;
+  }
```
Comment (RalucaRbc) [inline src/_component/_skills/skills.scss:13]: Add a new line here

File: src/_component/_skills/skills.scss [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+  &:nth-child(2) .skills__icon {
+    color: #50d166;
+  }
```
Comment (RalucaRbc) [inline src/_component/_skills/skills.scss:16]: and here

File: src/_component/_skills/skills.scss [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+  padding: 10px 12px 8px 10px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.skills__icon,
+.skills__label {
+  line-height: 48px;
+}
```
Comment (RalucaRbc) [inline src/_component/_skills/skills.scss:38]: And here

File: src/_component/_skills/skills.scss [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+  padding: 0 18px;
+  border-radius: 15px;
+  display: inline-block;
+  margin-right: 10px;
+  margin-bottom: 10px;
+}
+
+.skills__skill--heart {
+  background-color: #d52027;
+}
```
Comment (RalucaRbc) [inline src/_component/_skills/skills.scss:60]: And here

File: src/_component/_skills/skills.scss [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+.skills__skill--trophy {
+  background-color: #50d166;
+}
+
+.skills__skill--book {
+  background-color: #f18f1c;
+}
```
Comment (RalucaRbc) [inline src/_component/_skills/skills.scss:67]: And at the end of the file a new empty line

**Review by RalucaRbc — APPROVED** [review]

File: src/_component/_connect/connect.pug [hunk 7]
```diff
@@ -0,0 +1,48 @@
+-var sectionTitle= data.connect.sectionTitle
```
Comment (marianzburlea) [inline src/_component/_connect/connect.pug:1]: add a space after - 
`- var` and one before the = like:
` = `

File: src/_component/_connect/connect.pug [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+    .connect__avatar
+.column(span="row" height="10")
+
+
+each social, key in data.connect.socialList 
+  input.connect__radio(
+    type="radio"
+    name="connect__label"
+    id=`connect__label--${key}`
+    )
```
Comment (marianzburlea) [inline src/_component/_connect/connect.pug]: indent out (mai la stanga)

**Review by marianzburlea — CHANGES_REQUESTED** [review]

**Pull request merged**